docs/en/sql-reference/functions/string-functions.md

```diff
@@ -736,13 +734,13 @@ concat(s1, s2, ...)
 
 **Arguments**
 
-At least one value of arbitrary type.
+Values of arbitrary type. Empty argument is also allowed.
```

Comment:
Suggest to omit the second sentence, the first sentence implicitly assumes it (and it is a slightly silly special case without much practical relevance anyways).